tests/fixtures/transform_custom_dense.py

```diff
@@ -0,0 +1,14 @@
+import pandas as pd
```

Comment:
these files seem the same

Reply:
oh crap - you're right...I was converting to a sparse dataframe and then switched back and forgot it looked the same as the other one